hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/migration/AddBlockColumnsMigrationTest.java

```diff
@@ -0,0 +1,140 @@
+
+    private static final String REVERT_DDL =
+            """
+            alter table record_file
+              drop column round_end,
+              drop column round_start,
+              drop column software_version_major,
+              drop column software_version_minor,
+              drop column software_version_patch;
+            """;
+
+    private final @Owner JdbcTemplate jdbcTemplate;
+    private final RecordFileRepository recordFileRepository;
+
+    @Value("${migrationSql}")
+    private final Resource migrationSql;
+
+    @AfterEach
+    void cleanup() {
+        jdbcTemplate.execute(REVERT_DDL);
+    }
+
+    @Test
+    void empty() {
+        runMigration();
+        assertThat(recordFileRepository.findAll()).isEmpty();
+    }
+
+    @Test
+    void migrate() {
+        // given 3 record files all with unique hapi versions.
+        var expectedRecordFiles = List.of(
+                domainBuilder
+                        .recordFile()
+                        .customize(r -> r.hapiVersionMajor(1)
+                                .softwareVersionMajor(1)
+                                .roundEnd(null)
+                                .roundStart(null))
+                        .get(),
+                domainBuilder
+                        .recordFile()
+                        .customize(r -> r.hapiVersionMinor(2)
+                                .softwareVersionMinor(2)
+                                .roundEnd(null)
+                                .roundStart(null))
+                        .get(),
+                domainBuilder
+                        .recordFile()
+                        .customize(r -> r.hapiVersionPatch(3)
+                                .softwareVersionPatch(3)
+                                .roundEnd(null)
+                                .roundStart(null))
+                        .get());
+
+        // software versions are not included when converted to MigrationRecordFiles
+        expectedRecordFiles.forEach(this::persistRecordFile);
+
+        // when
+        runMigration();
+
+        // then the software versions are updated to equal the hapi versions
+        assertThat(recordFileRepository.findAll()).containsExactlyInAnyOrderElementsOf(expectedRecordFiles);
+    }
+
+    @SneakyThrows
+    private void runMigration() {
+        try (var is = migrationSql.getInputStream()) {
+            var script = StreamUtils.copyToString(is, StandardCharsets.UTF_8);
+            jdbcTemplate.execute(script);
+        }
+    }
+
+    static class Initializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+        @Override
+        public void initialize(ConfigurableApplicationContext configurableApplicationContext) {
+            var environment = configurableApplicationContext.getEnvironment();
+            String version;
+            String migrationSql;
+            if (environment.acceptsProfiles(Profiles.of("v2"))) {
+                version = "2.7.0";
```

Comment:
```suggestion
                version = "2.6.0";
```

for v1, version should be `1.101.0`

Because we want to config flyway to run migration to the version right before the migration script we to test.

Reply:
Makes sense, got it, fixed now.